express/scripts/martech.js

```diff
@@ -406,6 +407,19 @@ loadScript('https://www.adobe.com/marketingtech/main.min.js', () => {
 
   decorateAnalyticsEvents();
 
+  const { weight, id, random } = window.hlx.rum;
+  const generation = 'instrumentation-test-martech';
+  window.hlx.rum.generation = generation;
```

Comment:
Why do you need to set `window.hlx.rum.generation` ? You never re-use it.

Reply:
great catch, this one was actually wrong.